ansible/deploy_proxy.yml

```diff
@@ -18,7 +18,25 @@
     - { role: nginx, when: proxy_type == 'nginx' and active_sites.motech == True, action: site, site_name: motech}
     - { role: nginx, when: proxy_type == 'nginx' and active_sites.motech == True, action: site, site_name: motech_http }
     - { role: nginx, when: proxy_type == 'nginx' and active_sites.cchq_http_j2me == True, action: site, site_name: cchq_http_j2me }
-    - { role: nginx, when: proxy_type == 'nginx' and active_sites.riakcs == True, action: site, site_name: riakcs }
+    - role: nginx
+      action: site
+      site_name: riakcs
+      hosts_group: "{{ 'riakcs_new' if 'BLOB_DB_MIGRATING_FROM_S3_TO_S3' in localsettings else 'riakcs' }}"
+      when: proxy_type == 'nginx' and active_sites.riakcs == True
```

Comment:
moving the `riakcs_proxy_port` from `vars/all.yml` to here would make this easier to reason with